tests/end_to_end/features/steps/multifeature_registration.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+import featureform as ff
+from behave import given, when, then
+
+
+@when("I register postgres")
+def step_impl(context):
+    try:
+        context.postgres = ff.register_postgres(
+            name="postgres-quickstart",
+            host="172.17.0.1",  # The docker dns name for postgres
+            port="5432",
+            user="postgres",
+            password="password",
+            database="postgres",
+        )
+    except Exception as e:
+        context.exception = e
```

Comment:
We're setting this value but not checking it anywhere. Should remove the try/except statement